packages/nuxt/src/core/runtime/nitro/error.ts

```diff
@@ -74,15 +80,22 @@ export default <NitroErrorHandler> async function errorhandler (error: H3Error,
     return send(event, template(errorObject))
   }
 
-  const html = await res.text()
+  const html = await youch.toHTML(error, {
+    request: {
+      url,
+      method: event.method,
+      headers: getRequestHeaders(event),
+    },
+  })
+
   if (event.handled) { return }
 
```

Comment:
```suggestion
  if (event.handled) { return }
  
  const html = await youch.toHTML(error, {
    request: {
      url,
      method: event.method,
      headers: getRequestHeaders(event),
    },
  })
```

Reply:
Sorry for the lateness will push this change shortly :pray: